services/QuillLMS/client/app/bundles/Connect/components/studentLessons/fillInBlank.tsx

```diff
@@ -370,7 +389,18 @@ export class PlayFillInTheBlankQuestion extends React.Component<PlayFillInTheBla
     const { previewMode, question } = this.props;
     const { responses, inputErrors } = this.state
 
-    if (inputErrors && _.size(inputErrors) !== 0) {
+    const maxAttemptsSubmitted = question.attempts && question.attempts.length === ALLOWED_ATTEMPTS;
+    const latestAttempt = getLatestAttempt(question.attempts);
+
+    if (maxAttemptsSubmitted && !latestAttempt.response.optimal) {
+      const finalAttemptFeedback = `<b>Good try!</b> Compare your response to the strong response, and then go on to the next question.<br><br><b>Your response</b><br>${latestAttempt.response.text}<br><br><b>A strong response</b><br>${this.correctResponse()}`
```

Comment:
Since this code duplicates the `question.tsx` file, I think it would be best to extract the `ALLOWED_ATTEMPTS` constant and turn `finalAttemptFeedback` into a function that just takes the student's response and the correct response, and then import them here. That allows us to update fewer places in the event of a copy change.